launcher/ui/dialogs/CopyInstanceDialog.cpp

```diff
@@ -78,8 +78,13 @@ CopyInstanceDialog::CopyInstanceDialog(InstancePtr original, QWidget *parent)
     }
     ui->groupBox->setCurrentIndex(index);
     ui->groupBox->lineEdit()->setPlaceholderText(tr("No group"));
-    ui->copySavesCheckbox->setChecked(m_copySaves);
-    ui->keepPlaytimeCheckbox->setChecked(m_keepPlaytime);
+    ui->copySavesCheckbox->setChecked(m_selectedOptions.copySaves);
+    ui->keepPlaytimeCheckbox->setChecked(m_selectedOptions.keepPlaytime);
+    ui->copyGameOptionsCheckbox->setChecked(m_selectedOptions.copyGameOptions);
+    ui->copyResPacksCheckbox->setChecked(m_selectedOptions.copyResourcePacks);
+    ui->copyShaderPacksCheckbox->setChecked(m_selectedOptions.copyShaderPacks);
+    ui->copyServersCheckbox->setChecked(m_selectedOptions.copyServers);
+    ui->copyModsCheckbox->setChecked(m_selectedOptions.copyMods);
```

Comment:
maybe we could have getters / setters here, to further abstract away the internal data from that struct. may be a bit overkill though, so feel free to resolve this if you wish :p

Reply:
Okay, I'll start work on this soon

Reply:
Done! commit c00f96c.